src/verinfast/cloud/aws/costs.py

```diff
@@ -32,7 +32,16 @@ def _get_costs_and_usage(profile: str, aws_output_file: str):
         except subprocess.CalledProcessError:
             raise Exception("Error getting aws cli data.")
 
+        if results.returncode != 0:
```

Comment:
This can't happen. `CalledProcessError` will be thrown when check is True and the exit code is not 0.
https://docs.python.org/3/library/subprocess.html